### ISSUE TITLE
Validator details

### ISSUE THREAD

**Issue by csillag** [oasisprotocol/nexus#861]
mockups: [https://www.figma.com/file/8dwSyYOOm3vlgbvxPGT29p/Block-Explorer-(Post-Launch)?type=design&node-id=6092-212988&mode=design&t=y5RJp5Wyw0HndnoP-0](https://www.figma.com/file/8dwSyYOOm3vlgbvxPGT29p/Block-Explorer-\(Post-Launch\)?type=design&node-id=6092-212988&mode=design&t=y5RJp5Wyw0HndnoP-0)

current UI: not available until we switch query param, but it is mostly done

API endpoint: [https://nexus.oasis.io/v1/consensus/validators/9sAhd+Wi6tG5nAr3LwXD0y9mUKLYqfAbS2+7SZdNHB4=](https://nexus.oasis.io/v1/consensus/validators/9sAhd+Wi6tG5nAr3LwXD0y9mUKLYqfAbS2+7SZdNHB4=)

*   Many items relate to #859
*   use address instead of entity\_id as parameter (done in #685)
*   data for balance distribution chart (done in #686)
*   data to render "Voting Power" snapshot box (done in #686)
*   uptime (TODO: warren's #646) adds tracking block signers which can be used as a proxy for uptime. Once it's merged we'll need a follow-up PR to expose this on the validator endpoint)
*   media is empty in most cases #859
    *   (done in #707)
*   Missing data for details section:
    *   "Rank" (done in #686)
    *   "Delegators" (done in #686)
    *   "Status" in terms that mockups include waiting status
    *   "Start date" timestamp (wip in #686
    *   "Voting power" (done in #686)
    *   "Total share" (done in #686)
    *   "Participation rate" -- same as uptime, see above
*   API endpoint for Staking Trend chart (done in #703)
*   API endpoint for Signed Blocks, something like [https://www.oasisscan.com/mainnet/validator/stats?address=oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm](https://www.oasisscan.com/mainnet/validator/stats?address=oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm) (done in #760)
*   API endpoint for Proposed Blocks (wip in #764)

  

Undelegations [https://nexus.oasis.io/v1/consensus/accounts/oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v/debonding\_delegations\_to](https://nexus.oasis.io/v1/consensus/accounts/oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v/debonding_delegations_to)

*   debond\_end is always `0` , should we have something like epochLeft. How can I correctly show a date of debonding end?
^ this is a bug, @ptrus fixes it in #712
*   ~~How should we show shares. Oasis scan style:"76" or Wallet style (Gigashares~~**~~)~~** ~~76.060790809~~
    *   I believe this was resolved by frontend

## Subtasks:
- [x] #703 
- [x] #764 

**Update:**

@Andrew7234 wrote:

> Updated the task list with the current state! There are 3 remaining todos, which can likely all be completed in a follow-up PR once #646 (which tracks block signers/proposers) is merged. 
> 
> There is some more work to be done on validator status, which is currently impl as a `true/false`  but needs to be updated since there are 3 possible states of (active/inactive/waiting). Need some clarification on the difference between inactive/waiting. This will likely be impl in #686

**Comment by ptrus** [comment]
This was completed.
